app/assets/stylesheets/components/_external-link.scss

```diff
@@ -0,0 +1,7 @@
+.external-link {
+  @include display-icon('external-link', 'after', .65em, units(.5), '');
```

Comment:
if we're going with this visual treatment, could we just apply this style to `a[target=_blank]` and skip the partial approach?

Reply:
We could, but we'd have to override it to use the alternate white version for cases like the footer.

Reply:
Yeah that seems reasonable? Basically `footer a[target=_blank]` and be done with it?

Reply:
Done in de18674, but I'm not sure if the file should be renamed.

Reply:
Cool and now we can get rid of the partial entirely, right?

Reply:
We have to have both a visual notification and explicit text to let people using screen readers know the link will open a new window. The partial is needed for the latter part.

Reply:
Also scss-lint is mad about using the attribute selectors with an element?

Reply:
Hmmmm I have one more idea to help us get rid of a partial and leverage CSS.... we could also do the text as `::after { content: '' }` attribute

but we'd need to translate it per-local so we'd have to scope it by language. Luckily we sent `<html lang="en">` on every page, so:


```scss
html[lang=en] {
  a[target=_blank]::after {
    content: 'opens in a new window';
  }
}

html[lang=es] {
  a[target=_blank]::after {
    content: 'opens in a new window but in Spanish';
  }
}
```

I don't know about the scss-lint thoughj